### PR TITLE
doc: add reg members elected as voting members

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <michael_dawson@ca.ibm.com>

Looks like we missed adding this after the election closed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,13 @@ According to the [CPC Charter](https://github.com/openjs-foundation/cross-projec
 - Christopher Hiller (@boneskull, IBM)
 - Tobie Langel (@tobie, UnlockOpen)
 
+#### Elected by Regular Members
+According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), up to two regular
+members can be elected as votring members. The two regular members elected as voting members are currently:
+
+- Michael Dawson (@mhdawson)
+- Eemeli Aro (@eemeli)
```
Comment (mhdawson) [inline README.md:96]:
```suggestion
- Eemeli Aro (@eemeli)
- Michael Dawson (@mhdawson)
```

File: README.md [hunk 2]
```diff
@@ -88,6 +88,13 @@ According to the [CPC Charter](https://github.com/openjs-foundation/cross-projec
 - Christopher Hiller (@boneskull, IBM)
 - Tobie Langel (@tobie, UnlockOpen)
 
+#### Elected by Regular Members
+According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), up to two regular
+members can be elected as votring members. The two regular members elected as voting members are currently:
```
Comment (dmethvin) [inline README.md:93]:
```suggestion
members can be elected as voting members. The two regular members elected as voting members are currently:
```

**Comment by BesleagaAlexandruMarian** [comment]
Very good?!

File: README.md [hunk 3]
```diff
@@ -88,6 +88,13 @@ According to the [CPC Charter](https://github.com/openjs-foundation/cross-projec
 - Christopher Hiller (@boneskull, IBM)
 - Tobie Langel (@tobie, UnlockOpen)
 
+#### Elected by Regular Members
+According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), up to two regular
+members can be elected as voting members. The two regular members elected as voting members are currently:
+
+- Eemeli Aro (@eemeli)
+- Michael Dawson (@mhdawson)
```
Comment (tobie) [inline README.md:96]: For consistency:

```suggestion
- Eemeli Aro (@eemeli, Vincit)
- Michael Dawson (@mhdawson, IBM)
```

**Comment by mhdawson** [comment]
Lost of approvals, no objections and past required waiting time, landing.

**Pull request merged**